test/common/storagelib.py

```diff
@@ -588,11 +588,13 @@ def encrypt_root(self, passphrase):
 parted -s {dev} mkpart primary ext4 1M 500M
 parted -s {dev} mkpart primary ext4 500M 100%
 echo {passphrase} | cryptsetup luksFormat --pbkdf-memory=300 {dev}2
-echo {passphrase} | cryptsetup luksOpen --pbkdf-memory=300 {dev}2 dm-test
 luks_uuid=$(blkid -p {dev}2 -s UUID -o value)
-mkfs.ext4 /dev/mapper/dm-test
```

Comment:
Do we still test a direct "LUKS on plain physical disk" case somewhere? I know that these "encrypted root" tests are horribly expensive, and if we have to choose, then I agree we should test the more complex code path.

Reply:
> Or do you mean rootfs without LVM

Yes, that's what I meant.